pylutron/__init__.py

```diff
@@ -309,6 +309,10 @@ def _parse_area(self, area_xml, location):
           motion_sensor = self._parse_motion_sensor(device_xml)
           area.add_sensor(motion_sensor)
         #elif device_xml.get('DeviceType') == 'VISOR_CONTROL_RECEIVER':
+        else:
+          #phantom keypad doesn't have a DeviceType
```

Comment:
what specifically makes it a phantom keypad? This seems like a catch all and I'd rather us be explicit.

Reply:
They don't have a DeviceType
and also the led base is 2000.

<img width="1032" alt="Screenshot 2020-05-12 at 18 09 18" src="https://user-images.githubusercontent.com/29941924/81718167-b5c9a200-947b-11ea-8ac1-c585dc63628a.png">

Reply:
phantoms do not have device type... they also will have a serial # of "0" ...  Of course, you could insert any keypad in your project, and until you activate it, it will also have a "0" for a serial number, but the keypad you inserted would also have a device type... anyway for me with QS 14.5 I don't see any other situations where there is no device type, and a serial number of "0" .... just want to note these phantoms are made specifically for integration... leaving them out would be a big miss... That is the reason Lutron put them in there... they aren't for interacting with the Connect app, etc.. it doesn't even recognize them.. they are typically used to interact with Crestron, Control4, etc...